txn_writer.go

```diff
@@ -19,6 +19,16 @@ import (
 	"github.com/sdboyer/gps"
 )
 
+// Example string to be written to the manifest file
+// if no dependencies are found in the project
+// during `dep init`
+const exampleToml = `
+# Example:
```

Comment:
Can you add the remaining available options, `revision`, `version`, and `source` (e.g. https://github.com/myfork/package)? That way a user will know all the values that they can provide, and what an acceptable value looks like for each one.